packages/core/bootstrap/test/helpers/server.ts

```diff
@@ -46,19 +45,20 @@ export class Server {
       }
     })
 
-    this.app.get('/customError', (_, res) => {
+    app.get('/customError', (_, res) => {
       this.errorCount++
-      res.status(200).json({
+      res.status(500).send({
```

Comment:
Why are we changing the status code here?

Reply:
There has been an error here, I already solved it in the dev branch, we have a custom error for this test, but it had been deleted when migrating to fastify as a test.